service/controller/v11/templates/cloudformation/guest/iam_policies.go

```diff
@@ -37,11 +37,11 @@ const IAMPolicies = `{{define "iam_policies"}}
 
           - Effect: "Allow"
             Action: "s3:ListBucket"
-            Resource: "arn:aws:s3:::{{.S3Bucket}}"
+            Resource: "arn:{{.RegionARN}}:s3:::{{.S3Bucket}}"
```

Comment:
Do we need a migration path for this? I feel like an update from an old operator to this will fail, but I don't have much insight into aws-operator.

Reply:
Nope, as you can see in `key.RegionARN` in this same changeset `{{.RegionARN}}` will default to the previous value, `aws`.